lightningd/bitcoind.c

```diff
@@ -80,6 +80,9 @@ void bitcoind_check_commands(struct bitcoind *bitcoind)
 	for (i = 0; i < ARRAY_SIZE(methods); i++) {
 		p = find_plugin_for_command(bitcoind->ld, methods[i]);
 		if (p == NULL) {
+			/* For testing .. */
+			log_debug(bitcoind->ld->log, "Missing a Bitcoin plugin"
+						     " command");
```

Comment:
Should probably include the method name that can't be resolved.

Reply:
This is just used in functional tests, the `fatal` below includes it